contracts/Lisk_SpokePool.sol

```diff
@@ -4,12 +4,18 @@ import "@eth-optimism/contracts/libraries/constants/Lib_PredeployAddresses.sol";
 
 import "./Ovm_SpokePool.sol";
 import "./external/interfaces/CCTPInterfaces.sol";
+import { IOpUSDCBridgeAdapter } from "./external/interfaces/IOpUSDCBridgeAdapter.sol";
 
 /**
- * @notice Lisk Spoke pool.
+ * @notice Lisk SpokePool.
  * @custom:security-contact bugs@across.to
  */
 contract Lisk_SpokePool is Ovm_SpokePool {
+    using SafeERC20 for IERC20;
+
+    // Address of the custom L2 USDC bridge.
+    address public constant USDC_BRIDGE = 0x3b1ac69368eb6447F5db2d4E1641380Fa9e40d29;
```

Comment:
Work noting that I modified this from `private` to `public` - I don't see why we'd mark this as private; it's useful to be able to read it from the implementation contract.